RxSwift/Observable.swift

```diff
@@ -10,7 +10,7 @@
 ///
 /// It represents a push style sequence.
 public class Observable<Element> : ObservableType {
-    init() {
+    public init() {
```

Comment:
Required as there are various classes that derive from Observable that have a public init method, and the generated .swiftinterface files don't like this.
Seems like it probably wants to be public anyway as the class is public

Reply:
This is actually a bit of an unwanted side-effect. It would be preferable to leave the initializer internal in this case. Is there any way to do this anyways? There is no value in initializing Observable directly this way.

Reply:
discussion in DisposeBase.swfit